tutorials/01_introductory/sample_plots.py

```diff
@@ -55,11 +57,11 @@
 Path demo
 =========
 
-You can add arbitrary paths in matplotlib using the
+You can add other paths in matplotlib using the
```

Comment:
Also needs to be arbitrary cause the arbitrary is in reference to the shape of the part (line, circle, curve)